multiregion/docs/multiregion.rst

```diff
@@ -112,6 +112,13 @@ Topic
 -  ``--replica-placement <path-to-replica-placement-policy-json>``: at
    topic creation, this argument defines the replica placement policy for a given
    topic
+   
+Docker commands
+---------------
+
+This version of the tutorial shows ``docker compose`` commands. Earlier versions of the Docker Compose CLI
```

Comment:
"Earlier versions of the Docker Compose CLI" intended to be "Earlier versions of the tutorial"?